extensions/data-transfer/portability-data-transfer-apple/src/main/java/org/datatransferproject/datatransfer/apple/photos/AppleMediaInterface.java

```diff
@@ -566,6 +571,12 @@ Map<String, Long> importMediaBatch(
         String mediaType = downloadableFile.getMimeType();
         String albumId = downloadableFile.getFolderId();
         Long creationDateInMillis = getUploadedTime(downloadableFile);
+        Boolean isFavorite = false;
```

Comment:
can we make this a `boolean` and also may be treat the potential null value as `falsy`? So that as @jzacsh suggested the c'tor can take `isFavorite` as boolean ?